index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  <title>My Portfolio</title>
+</head>
+<body>
+
+  <header>
+
+    <div id="settings">
+      <div id="settings-date">
+
+      </div>
+      
+      <div class="settings-fa">
+        <i class="fas fa-signal"></i>
+        <i class="fas fa-wifi"></i>
+        <i class="fas fa-battery-full"></i>
+      </div>
+    </div>
```

Comment:
- [ ] Nice work sticking to the Figma template (literally 😄 ), but this part is a phone screen bar and is not part of the design, please make sure to take it off.  
![2023-03-23_231127](https://user-images.githubusercontent.com/104319462/227385127-9c62ac11-8219-41e0-98d4-c349f7cb1048.png)

Reply:
😄😄😄 Done, thanks @ouasamine, And saying that it took me a lot of time to search them... 😄😄😄